lib/devise/multi_email/models/authenticatable.rb

```diff
@@ -29,41 +29,73 @@ module AuthenticatableExtensions
         extend ActiveSupport::Concern
 
         included do
+          multi_email_association.configure_autosave!{ include AuthenticatableAutosaveExtensions }
```

Comment:
Done (although I prefer without a space so it doesn't look like a Hash being passed to the method).